lib/winapi/winbase.h

```diff
@@ -93,6 +94,8 @@ void SetLastError (DWORD error);
 
 void WINAPI OutputDebugStringA (LPCTSTR lpOutputString);
 
+BOOL IsBadWritePtr(LPVOID lp, UINT_PTR ucb);
```

Comment:
nit: Other winapi functions have a space between function name and parameter parentheses (same as above).

Reply:
Good catch - I assume this affects all my PRs from today.
Should have been fixed here, I'll fix the rest later.